src/annotator/test/pdf-sidebar-test.js

```diff
@@ -0,0 +1,238 @@
+import $ from 'jquery';
+
+//import events from '../../shared/bridge-events';
+
+import PdfSidebar from '../pdf-sidebar';
+import { $imports } from '../pdf-sidebar';
+
+describe('PdfSidebar', () => {
+  const sandbox = sinon.createSandbox();
+  let CrossFrame;
+  let fakeCrossFrame;
+
+  let fakePDFViewerApplication;
+  let fakePDFContainer;
+  let fakePDFViewerUpdate;
+  const sidebarConfig = { pluginClasses: {} };
+
+  const createPdfSidebar = config => {
+    if (!config) {
+      config = {};
+    }
+    config = Object.assign({}, sidebarConfig, config);
+    const element = document.createElement('div');
+    return new PdfSidebar(element, config);
+  };
+
+  beforeEach(() => {
+    sandbox.stub(PdfSidebar.prototype, '_setupGestures');
+    fakePDFContainer = document.createElement('div');
+    fakePDFViewerUpdate = sinon.stub();
+
+    fakePDFViewerApplication = {
+      appConfig: {
+        appContainer: fakePDFContainer,
+      },
+      pdfViewer: {
+        currentScaleValue: 'auto',
+        update: fakePDFViewerUpdate,
+      },
+    };
+    // See https://github.com/sinonjs/sinon/issues/1537
+    // Can't stub an undefined property in a sandbox
+    window.PDFViewerApplication = fakePDFViewerApplication;
+
+    // From `Sidebar.js` tests
+    fakeCrossFrame = {};
+    fakeCrossFrame.onConnect = sandbox.stub().returns(fakeCrossFrame);
+    fakeCrossFrame.on = sandbox.stub().returns(fakeCrossFrame);
+    fakeCrossFrame.call = sandbox.spy();
+    fakeCrossFrame.destroy = sandbox.stub();
+
+    const fakeBucketBar = {};
+    fakeBucketBar.element = $('<div></div>');
+    fakeBucketBar.destroy = sandbox.stub();
+
+    CrossFrame = sandbox.stub();
+    CrossFrame.returns(fakeCrossFrame);
+
+    const BucketBar = sandbox.stub();
+    BucketBar.returns(fakeBucketBar);
+
+    sidebarConfig.pluginClasses.CrossFrame = CrossFrame;
+    sidebarConfig.pluginClasses.BucketBar = BucketBar;
+  });
+
+  afterEach(() => {
+    delete window.PDFViewerApplication;
+    sandbox.restore();
+    $imports.$restore();
```

Comment:
Actually, I don't think we need `$imports` at all, I think it's not being mocked right?